server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java

```diff
@@ -775,6 +775,7 @@ public void testEmptyShards() throws IOException, InterruptedException {
         assertAcked(prepareCreate("test").setSettings(Settings.builder()
                 .put(indexSettings())
                 .put(IndexSettings.MAX_SHINGLE_DIFF_SETTING.getKey(), 4)
+                .put("index.refresh_interval", -1)  // prevents occasional scoring glitches due to multi segments
```

Comment:
@cbuescher I am not completely clear how multiple segments should change scoring. I thought scores should be the same regardless of number of segments.  Should we instead change the number of shards in the index to be `1`? 
But I am also in favour of completely removing the last part of this test with documents, as the test is about empty shards

Reply:
Its not about scoring (as in search scoring) per se here. The way WordScorer uses the term frequencies of an index reader seems to differ depending on how many leaves it has, at least that is what Simon told me. I don't pretend to understand the subtleties here. I wouldn't like to change the random number of shards here because I think that coverage is important here, but turning of refresh is fine I think and what we seem do to in a lot of places (just grepped and found about 36 cases of this in tests).

Reply:
@cbuescher Thanks for explanation.